tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md

```diff
@@ -32,11 +32,6 @@ This tutorial uses billable components of Google Cloud, including Compute Engine
 Use the [pricing calculator](https://cloud.google.com/products/calculator/) to generate a cost estimate based on your projected usage. New Google Cloud
 users might be eligible for a [free trial](https://cloud.google.com/free-trial).
 
-If you follow this tutorial, the costs are estimated as follows, depending on the operating system that you use:
-
-* [Linux VMs](https://cloud.google.com/products/calculator#id=ca135004-465c-4a43-bc5b-701af07df644)
```

Comment:
@ToddKopriva The following link works for me, could you do a check again.

Reply:
Yes, I'll check again as I make my second pass.